### PR TITLE
Fix pulling and placing skulls/heads from creative inventory

### PR DESCRIPTION
**Pull request by Tjorven-Liebe**
Camotoys fix regarding the skulls and heads is not really clean I think so I tried to do it properly.
Needs mapping pr to also be updated. see [Mappings](https://github.com/GeyserMC/mappings/pull/98)

### CODE REVIEW

File: core/src/main/java/org/geysermc/geyser/registry/populator/Conversion748_729.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+public class Conversion748_729 {
+
+    private static final Map<String, Integer> NEW_PLAYER_HEADS = Map.of("minecraft:skeleton_skull", 0, "minecraft:wither_skeleton_skull", 1, "minecraft:zombie_head", 2, "minecraft:player_head", 3, "minecraft:creeper_head", 4, "minecraft:dragon_head", 5, "minecraft:piglin_head", 6);
+
+    static GeyserMappingItem remapItem(Item item, GeyserMappingItem mapping) {
+        String identifier = mapping.getBedrockIdentifier();
+
+        if (NEW_PLAYER_HEADS.containsKey(identifier)) {
+            return mapping.withBedrockIdentifier("minecraft:skull")
+                .withBedrockData(0);
```
Comment (onebeastchris) [inline core/src/main/java/org/geysermc/geyser/registry/populator/Conversion748_729.java:42]: Shouldn't this return the according bedrock data from the map?

**Review by onebeastchris — APPROVED** [review]

File: core/src/main/java/org/geysermc/geyser/registry/populator/Conversion748_729.java (hunk 1 above)
Comment (Tjorven-Liebe) [reply core/src/main/java/org/geysermc/geyser/registry/populator/Conversion748_729.java:42]: yes it should

**Comment by onebeastchris** [comment]
Merged the mappings changes - once this PR updates the mappings submodule, I'll merge this PR too. Thanks for cleaning this up!

**Comment by Tjorven-Liebe** [comment]
> Merged the mappings changes - once this PR updates the mappings submodule, I'll merge this PR too. Thanks for cleaning this up!

no problem!

**Comment by Tjorven-Liebe** [comment]
@onebeastchris done!

**Review by onebeastchris — APPROVED** [review]

**Comment by onebeastchris** [comment]
Thanks!

**Pull request merged**